pkg/kubectl/cmd/create.go

```diff
@@ -146,7 +146,7 @@ func printObjectSpecificMessage(obj runtime.Object, out io.Writer) {
 func makePortsString(ports []api.ServicePort) string {
 	pieces := make([]string, len(ports))
 	for ix := range ports {
-		pieces[ix] = fmt.Sprintf("%s:%d", strings.ToLower(string(ports[ix].Protocol)), ports[ix].Port)
+		pieces[ix] = fmt.Sprintf("%s:%d", strings.ToLower(string(ports[ix].Protocol)), ports[ix].NodePort)
```

Comment:
This is right for one caller of this function but not for LoadBalancers.  Probably need to pass a function pointer that takes a *api.ServicePort and returns either Port or Node Port